src/Microsoft.VisualBasic/ref/Microsoft.VisualBasic.cs

```diff
@@ -201,6 +202,11 @@ public sealed partial class VBMath
         public static float Rnd() { throw null; }
         public static float Rnd(float Number) { throw null; }
     }
+    //[Microsoft.VisualBasic.CompilerServices.StandardModuleAttribute]
+    //internal sealed partial class Interaction
+    //{
+    //    internal Interaction() { }
+    //}
```

Comment:
Is `ref/Microsoft.VisualBasic.cs` a generated file?

Reply:
There is/was an (internal?) utility that has been used to generate part of it originally but since then people manually edit it. I have never used it.

Reply:
Thanks. Please remove the commented out code.

Reply:
This covered a bug, reported separately. the Interaction class should be public as it is in the reference. I will delete.